astroplan/plots/sky.py

```diff
@@ -0,0 +1,195 @@
+    altitude = (91 * u.deg - observer.altaz(time, target).alt) *(1/u.deg)
+    # Azimuth MUST be given to plot() in radians.
+    azimuth = observer.altaz(time, target).az * (1/u.deg) * (np.pi/180.0)
+
+    # Some checks & info for labels.
+    if not hasattr(target, 'name'):
+        target_name = '?'
+    else:
+        target_name = target.name
+    style_kwargs.setdefault('label', target_name)
+
+    # We only want to plot positions above the horizon.
+    az_plot = None
+    for alt in range(0, len(altitude)):
+        if altitude[alt] > 91.0:
+            print("Warning: Target " + str(target_name) +
+                  " is below the horizon at time: " + str(time[alt]))
+        else:
+            if az_plot is None:
+                az_plot = np.array([azimuth[alt]])
+            else:
+                az_plot = np.append(az_plot, azimuth[alt])
+    alt_plot = altitude[altitude <= 91.0]
+    if az_plot is None:
+        az_plot = []
+
+    # More axes set-up.
+    # Position of azimuth = 0 (data, not label).
+    ax.set_theta_zero_location('N')
+
+    # Direction of azimuth increase. Clockwise is -1
+    if north_to_east_ccw is False:
+        ax.set_theta_direction(-1)
+
+    # Plot target coordinates.
+    # ax.plot connects dots.
+    #ax.plot(az_plot, alt_plot, **style_kwargs)
+    # ax.scatter does not connect dots.
+    ax.scatter(az_plot, alt_plot, **style_kwargs)
+    #ax.polar(az_plot, alt_plot, **style_kwargs)
+
+    # Set radial limits.
+    ax.set_rlim(1, 91)
+
+    # Grid, ticks & labels.
+    # May need to set ticks and labels AFTER plotting points.
+    if grid is True:
+        ax.grid(True, which='major')
+    if grid is False:
+        ax.grid(False)
+    degree_sign = u'\N{DEGREE SIGN}'
+
+    # For positively-increasing range (e.g., range(1, 90, 15)),
+    # labels go from middle to outside.
+    r_labels = [
+        '90' + degree_sign,
+        '',
+        '60' + degree_sign,
+        '',
+        '30' + degree_sign,
+        '',
+        '0' + degree_sign + ' Alt.',
+        ]
+
+    theta_labels = []
+    for chunk in range(0, 7):
+        label_angle = (az_label_offset*(1/u.deg)) + (chunk*45.0)
+        while label_angle >= 360.0:
+            label_angle = label_angle - 360.0
+        if chunk == 0:
+            theta_labels.append('N ' + '\n' + str(label_angle) + degree_sign
+                                + ' Az')
+        elif chunk == 2:
+            theta_labels.append('E' + '\n' + str(label_angle) + degree_sign)
+        elif chunk == 4:
+            theta_labels.append('S' + '\n' + str(label_angle) + degree_sign)
+        elif chunk == 6:
+            theta_labels.append('W' + '\n' + str(label_angle) + degree_sign)
+        else:
+            theta_labels.append(str(label_angle) + degree_sign)
+
+    # Set ticks and labels.
+    ax.set_rgrids(range(1, 106, 15), r_labels, angle=-45)
+    ax.set_thetagrids(range(0, 360, 45), theta_labels, frac=1.2)
+
+    # Below commands don't seem to work.
```

Comment:
@eteq: I'd like to leave this leftover code here as a warning to future contributors that these particular commands don't work with the code as is--would that be ok?